app/models/user.rb

```diff
@@ -20,5 +20,43 @@ def name_for(id)
         I18n.t('values.deleted_user_name')
       end
     end
+
+    #
+    # TODO find a better home.
+    #
```

Comment:
The common term is "authenticate" so I guess that tells us something! :) 

We could have an authentication service etc.